cellprofiler/modules/tests/test_identifyprimaryobjects.py

```diff
@@ -2279,32 +2279,32 @@ def test_09_01_small_images(self):
                 l1, g1 = x.get_threshold(cpi.Image(image), mask, workspace)
                 self.assertAlmostEqual(l1, l)
 
-    def test_09_02_mog_fly(self):
-        """Test mixture of gaussians thresholding on the fly image"""
-        image = fly_image()
-        workspace, x = self.make_workspace(image)
-        x.threshold_method.value = T.TM_MOG
-        x.threshold_scope.value = I.TS_GLOBAL
-        x.object_fraction.value = '0.10'
-        local_threshold,threshold = x.get_threshold(
-            cpi.Image(image), np.ones(image.shape,bool), workspace)
-        self.assertTrue(threshold > 0.040)
-        self.assertTrue(threshold < 0.050)
-        x.object_fraction.value = '0.50'
-        local_threshold,threshold = x.get_threshold(
-            cpi.Image(image), np.ones(image.shape,bool), workspace)
-        self.assertTrue(threshold > 0.0085)
-        self.assertTrue(threshold < 0.0090)
+    # def test_09_02_mog_fly(self):
```

Comment:
Fine for now as we talked about previously - would be solved if we added the fly image as a resource similarly to the test image in Centrosome (tests/resources/kirsch.tiff). But that is a lower priority than competing the setup.